ingest/expression_files/dense_ingestor.py

```diff
@@ -0,0 +1,179 @@
+"""Module for ingesting dense matrix files
+
+DESCRIPTION
+Module provides extract and transforms function for gene expression data for
+an dense matrix.
+
+PREREQUISITES
+Must have python 3.6 or higher.
```

Comment:
Noting prerequisites in `ingest_pipeline.py` makes sense; less so in modules that can't (or wouldn't) be run via CLI.

```suggestion
```

(This also reduces maintenance burden, e.g. we note the updated requirement for Python 3.7 in fewer places.)